metricbeat/autodiscover/builder/hints/metrics.go

```diff
@@ -56,6 +58,17 @@ type metricHints struct {
 	Registry *mb.Register
 }
 
+type moduleCfg common.MapStr
+
+func (mcfg *moduleCfg) SecureString() string {
+	mcfgCopy := common.MapStr(*mcfg).Clone()
+	if exists, _ := mcfgCopy.HasKey(mcfgPassword); exists {
```

Comment:
check `/libbeat/common/config.go` (`DebugString`, `PrintDebugf`). There we have a blacklist of keys we don't want to show.

Reply:
Updated code to use `common.DebugString`.